_2_oo/src/main/java/code/_3_in_class/Main.java

```diff
@@ -3,6 +3,6 @@
 public class Main {
```

Comment:
Buna Corina,

Observ ca tu ai scris codul in folderul cu numele "_2_oo" care este destinat saptamanii 2 de curs cu tema Object Oriented.
Te rog sa creezi o clasa nou "HelloWorld" in pachedul corespunzator din folderul "_1_basics".